File: BUBBLE SORT/BUBBLESORT.py
def bubble_sort(arr):
    n = len(arr)
    for i in range(n - 1):
        for j in range(n - 1 - i):
            if arr[j] > arr[j + 1]:
                arr[j], arr[j + 1] = arr[j + 1], arr[j]

# Ejemplo súper simple
datos = [1, 4, 2, 3,5,8,6]
print("Arreglo sin ordenar =", datos)

bubble_sort(datos)

print("Arreglo ordenado   =", datos)

File: BUBBLE SORT/BUBBLESORT.cpp
#include <iostream>
using namespace std;

void bubbleSort(int arr[], int n) {
    for (int i = 0; i < n - 1; i++) {
        for (int j = 0; j < n - 1 - i; j++) {
            if (arr[j] > arr[j + 1]) {
                int tmp = arr[j];
                arr[j] = arr[j + 1];
                arr[j + 1] = tmp;
            }
        }
    }
}

int main() {
    int datos[] = {1, 4, 2, 3};
    int n = sizeof(datos) / sizeof(datos[0]);

    cout << "Arreglo sin ordenar = ";
    for (int i = 0; i < n; i++) cout << datos[i] << " ";
    cout << endl;

    bubbleSort(datos, n);

    cout << "Arreglo ordenado   = ";
    for (int i = 0; i < n; i++) cout << datos[i] << " ";
    return 0;
}
